src/main.cpp

```diff
@@ -162,7 +163,8 @@ int main(int argc, char** argv)
                 "Optional inputs:\n"
                 "\t -i <injection pid>        The PID of the process to hijack for injection\n"
                 "\t -I <injection thread>     The ThreadID in the process to hijack for injection (requires -i)\n"
-                "\t -e <inject_exe>           The executable to start with injection\n"
+                "\t -e <inject_file>          The executable to start with injection\n"
+                "\t -m <inject_method>        The injection method (default or shellexec for Windows amd64 only)\n"
```

Comment:
Instead of `default`, this should say `createproc`

Reply:
@tklengyel, if You read code more careful You could notice that this have been copied from Yours "-o" option which accepts "kv" and "csv" but names "kv" as default in help.

Reply:
The difference is that you actually have `!strncmp(optarg,"createproc",10)`, so the string `createproc` has a specific meaning. This should be reflected in the help text as well.